### DISCUSSION TITLE
Making on app public out of a private monorepo

### DISCUSSION THREAD

**Discussion by IlirEdis** [vercel/turborepo#5306] [Help]
My use case is: A main app (nextjs) and another one (Nextra) for the docs.
I want my main app code to be a private repo but i want to make the /docs app to be public so i can share it with other coworkers which will be responsible to make the edits on our docs.
How can i achieve this?
Thanks!

**Comment by mehdi-cit** [comment]
Hi @IlirEdis 
I just got started with Turbo repo and I have a similar question. Some of the projects within the monorepo I want to build need to be open for public (so people can inspect the code on Github) whereas other projects are closed/private.
All projects though need to be published to npm for people to use but not all have their code directly accessible in Github.
Any luck with satisfying your requirements? Help would be much appreciated
